packages/strapi-admin/admin/src/translations/sv.json

```diff
@@ -0,0 +1,157 @@
+{
+  "Analytics": "Analytics",
+  "Content Manager": "Innehållshantering",
+  "Content Type Builder": "Innehållstyp Byggare",
```

Comment:
This is a weird translation. "Innehållstypsmodellering" or something might make more sense. "Konstruering" is another option. "Byggare" is pretty much only used in construction/carpenting.

Reply:
So which translation is the best?

Reply:
`"Innehållstypsverktyg"` or even clearer `"Konstruera innehållstyper"`. Sometimes it's better to completely rewrite the translation so it make sense in the target language.

Reply:
Committed. 

> `"Innehållstypsverktyg"` or even clearer `"Konstruera innehållstyper"`. Sometimes it's better to completely rewrite the translation so it make sense in the target language.
> 
> `"Konstruera innehållstyper"` would make it clear to me what this is about.